### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by ioistired** [tallbl0nde/PlayEventParser#1]
Code is not open source without a license. Please add one. I suggest the [Blue Oak Model License 1.0.0](https://blueoakcouncil.org/license/1.0.0) as it is a simple permissive license that still covers all the bases (patent grant, language that includes all contributors, rather than just the author, path to forgiveness in case of violation) and it doesn't require copying the entire license, just a link to it.

**Comment by ioistired** [comment]
Or you could use the GPLv3 as it is a user-facing application.

**Comment by tallbl0nde** [comment]
Done, I didn't realise it didn't have one :P

**Comment by ioistired** [comment]
tfw worse license than MIT

**Comment by tallbl0nde** [comment]
Is it? The code is just open source so people from the scene can't go "there's no source, does this brick my Switch?" I don't really care what people do with it. I'm happy to change it to MIT but I figured this license achieves the same thing.

**Comment by ioistired** [comment]
MIT achieves roughly the same thing, including in countries that don't honor public domain (e.g. Germany). ISC does the same thing as MIT, but in fewer words. BlueOak does the same thing as ISC, but in plain english, including a patent grant (which is important even if you don't have any patents), and does not require attribution.

**Comment by tallbl0nde** [comment]
Ah right, well thanks for letting me know! I didn't know about the whole public domain thing. I'll change it to MIT as I'm familiar with it then.
